client/app/kubernetes/service/terminate/kubernetes-service-terminate.modal.controller.js

```diff
@@ -0,0 +1,29 @@
+class kubernetesTerminateModalCtrl {
+  constructor($scope, $state, $stateParams, CucControllerHelper) {
+    this.$scope = $scope;
+    this.$state = $state;
+    this.serviceName = $stateParams.serviceName;
+    this.CucControllerHelper = CucControllerHelper;
+    this.openModal();
+  }
+
+  openModal() {
+    this.CucControllerHelper.modal.showModal({
+      modalConfig: {
+        templateUrl: 'app/kubernetes/service/terminate/kubernetes-service-terminate.html',
```

Comment:
Maybe you could `import template` instead and just 
```suggestion
        template,
```